code/game/objects/items/tanks/watertank.dm

```diff
@@ -720,3 +720,40 @@
 	reagents.trans_to(user,used_amount,multiplier=usage_ratio)
 	update_icon()
 	user.update_inv_back() //for overlays update
+
+/datum/action/item_action/activate_injector
+	name = "Activate Injector"
+
+//Operator backpack spray
+/obj/item/watertank/op
+	name = "backpack water tank"
+	desc = "A New Russian backpack spray for systematic cleansing of carbon lifeforms."
+	icon_state = "waterbackpackop"
+	item_state = "waterbackpackop"
+	w_class = WEIGHT_CLASS_NORMAL
+	volume = 2000
+	slowdown = 0
+
+/obj/item/watertank/op/Initialize(mapload)
+	. = ..()
+	reagents.add_reagent(/datum/reagent/toxin/mutagen,350)
+	reagents.add_reagent(/datum/reagent/napalm,125)
+	reagents.add_reagent(/datum/reagent/fuel,125)
+	reagents.add_reagent(/datum/reagent/clf3,300)
+	reagents.add_reagent(/datum/reagent/cryptobiolin,350)
+	reagents.add_reagent(/datum/reagent/toxin/plasma,250)
+	reagents.add_reagent(/datum/reagent/consumable/condensedcapsaicin,500)
+
+/obj/item/reagent_containers/spray/mister/op
+	desc = "A mister nozzle attached to several extended water tanks. It suspiciously has a compressor in the system and is labelled entirely in New Cyrillic."
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "misterop"
+	item_state = "misterop"
+	lefthand_file = 'icons/mob/inhands/equipment/mister_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/mister_righthand.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	amount_per_transfer_from_this = 100
+	possible_transfer_amounts = list(75,100,150)
+
+/obj/item/watertank/op/make_noz()
+	return new /obj/item/reagent_containers/spray/mister/op(src)
```

Comment:
This seems like a random feature being added from TG that isn't relevant to the port

Reply:
sorry didnt notice